packages/react-popover/src/utils.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+// Need to disable compilation for aliases: https://github.com/microsoft/fluentui/pull/16976/files#r575447074
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button as ReactButton, ButtonProps } from '@fluentui/react-button';
+
+export const ExampleContent = () => {
+  return (
+    <div>
+      <h3>Popover content</h3>
+
+      <div>This is some popover content</div>
+    </div>
+  );
+};
```

Comment:
Usage of `h3` there produces weird margin:

![image](https://user-images.githubusercontent.com/14183168/132212424-a6ab1f37-45c4-4ac5-b7da-e3e6e49cd507.png)

Reply:
yeah user agent stylesheet, added a makestyles override